plugins/lookup/github_app_access_token.py

```diff
@@ -0,0 +1,156 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2023, Poh Wei Sheng <weisheng-p@hotmail.sg>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = '''
+    name: github_app_access_token
+    author:
+      - Poh Wei Sheng (@weisheng-p)
+    short_description: Obtain short-lived Github App Access tokens
+    version_added: '8.2.0'
+    requirements:
+      - jwt (https://github.com/GehirnInc/python-jwt)
+    description:
+      - This generates a Github access token that can be used with a C(git) command, if you use a Github App.
+    options:
+      key_path:
+        description:
+        - Path to your private key.
+        required: true
+        type: path
+      app_id:
+        description:
+        - Your GitHub App ID, you can find this in the settings page.
+        required: true
+        type: str
+      installation_id:
+        description:
+        - The installation id that contains the git repo you would like access to
+        - As of 2023-12-24, this can be found via settings page > integrations > application. The last part of the url in the
+          configure button is the installation id.
+        - Alternatively, you can use PyGithub (https://github.com/PyGithub/PyGithub) to get your installation id
```

Comment:
```suggestion
        - The installation ID that contains the git repository you would like access to.
        - As of 2023-12-24, this can be found via Settings page > Integrations > Application. The last part of the URL in the
          configure button is the installation ID.
        - Alternatively, you can use PyGithub (U(https://github.com/PyGithub/PyGithub)) to get your installation ID.
```